src/main/java/jmgomez/apipolicy/service/PolicyService.java

```diff
@@ -34,6 +34,6 @@ public Accident[] getAccidentsByPolicies(String id) {
 
     @Override
     public Accident getAccidentByPolicies(String policyId, String accidentId) {
-        return policyClient.getAccidentByPolicies("12345" + policyId, "A12345" + accidentId);
+        return policyClient.getAccidentByPolicies("12345" + policyId, accidentId);
```

Comment:
So your source of information is a pattern you've seen in the mock service. What will happen when we switch from the mock server to the actual endpoint?  Coupling your logic to mock data is very wrong. Your source of truth comes from the API itself, not the data you see in a test/mock file